### PR TITLE
First release

### PR DESCRIPTION
**Pull request by amon-cofie**
This is the first release of my air pollution tracking app for my capstone project. I did the following things in this project:
- Added the homepage which displays a list of citys when the page loads
- Added the details page which displays the air pollution data and air quality index of a selected city
- Added City.js component to render each city with its details
- Added middleware to get data from 3 APIs: 

> geolocation API
> air pollution API
> rest countries API

- Added logic functions to score each pollute concentration
- Added tests for components and functions
- Deployed on netlify

### CODE REVIEW

File: src/components/DataPage.js [hunk 1]
```diff
@@ -0,0 +1,124 @@
...
+            <tr>
+              <td>
+                SO
+                <sub>2</sub>
+              </td>
+              <td>{pollutes.so2}</td>
+              <td>{scoreSO2(pollutes.so2)}</td>
+            </tr>
+          </tbody>
+        </table>
```
Comment (ShadyShawkat) [inline src/components/DataPage.js:117]: - [ ]  Kindly stick as much as possible to the original design. On mobile view (375px), the background color of the table doesn't cover all of the table and causes the following visual bug which makes it harder to read the last couple of rows. Kindly fix this.
image

**Review by ShadyShawkat — CHANGES_REQUESTED** [review]
## Hi @amon-cofie, 

Good job so far! 👍🏻 
There are some issues that you still need to work on to go to the next project but you are almost there!

## To Highlight
- All linters are passing ✔️ 
- Correct use of GitFlow ✔️ 
- Descriptive commit messages ✔️ 
- Professional looking pull request ✔️ 

Image

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/components/DataPage.js (hunk 1 above)
Comment (amon-cofie) [reply src/components/DataPage.js:117]: Thank you very much, I will solve this shortly

File: README.md [hunk 2]
```diff
@@ -1,70 +1,89 @@
...
+## Live Demo
 
-The page will reload when you make changes.\
-You may also see any lint errors in the console.
+![Homepage](./src/resources/images/Homepage.png) ![Details](./src/resources/images/Details.png)
+![preview](./src/resources/images/preview.gif)
 
-### `npm test`
+- [Live Demo Link](https://airpollutiontracker.netlify.app/)
+- [Loom Presentation Video Link](https://www.loom.com/share/6bb9836479b147799efb9e2f413828f4)
```
[the remark this answers is not in the document]
Comment (amon-cofie) [reply README.md:19]: Thank you. I will do that

**Review by thecodechaser — APPROVED** [review]
## STATUS: APPROVED 🟢

## Hi @amon-cofie👋, 

### You did a marvelous job on the React Capstone project and this is one of the best projects I have seen so far!
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## To Highlight 🎉
- No linter errors. :heavy_check_mark:
- Gitflow is followed. :heavy_check_mark:
- Professional README file. :heavy_check_mark:
- PR has a good title and summary. ✔️
- All tests are passed. ✔️



## Optional suggestions

- [ ] I would suggest using this new Microverse README [template ](https://github.com/microverseinc/readme-template), It will make the README file looks more professional.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**